src/frontend/js/components/StepBreadcrumb/index.spec.tsx

```diff
@@ -0,0 +1,170 @@
+    // only a breadcrumb with the step index is displayed.
+    const manifest = {
+      start: 'step0',
+      steps: {
+        step0: {
+          next: 'step1',
+        },
+        step1: {
+          next: null,
+        },
+      },
+    };
+
+    const { result } = renderHook(() => useStepManager(manifest));
+    const { container, rerender } = render(
+      <StepBreadcrumb manifest={manifest} step={result.current.step} />,
+    );
+
+    expect(getAllByRole(container, 'listitem')).toHaveLength(2);
+
+    const labels = container.querySelectorAll('h6.StepBreadcrumb__step__label');
+    const stepsIcons = container.querySelectorAll('div.StepBreadcrumb__step__icon > *');
+    let activeSteps = container.querySelectorAll(
+      'li.StepBreadcrumb__step.StepBreadcrumb__step--active',
+    );
+    let currentStep = container.querySelector(
+      'li.StepBreadcrumb__step.StepBreadcrumb__step--current',
+    );
+    expect(labels).toHaveLength(0);
+    expect(stepsIcons).toHaveLength(2);
+    stepsIcons.forEach((icon, index) => {
+      expect(icon).toBeInstanceOf(HTMLSpanElement);
+      expect(icon).toHaveTextContent((index + 1).toString());
+    });
+    expect(activeSteps).toHaveLength(1);
+    expect(currentStep).toHaveTextContent('1');
+
+    act(() => result.current.next());
+    rerender(<StepBreadcrumb manifest={manifest} step={result.current.step} />);
+
+    activeSteps = container.querySelectorAll(
+      'li.StepBreadcrumb__step.StepBreadcrumb__step--active',
+    );
+    currentStep = container.querySelector('li.StepBreadcrumb__step.StepBreadcrumb__step--current');
+    expect(activeSteps).toHaveLength(2);
+    expect(currentStep).toHaveTextContent('2');
+  });
+
+  it('renders visually a complete manifest', () => {
+    // If manifest's steps has `label` and `icon` property,
+    // these information are shown.
+    const manifest = {
+      start: 'step0',
+      steps: {
+        step0: {
+          label: '0. Step',
+          icon: '#icon-0',
+          next: 'step1',
+        },
+        step1: {
+          label: '1. Step',
+          icon: '#icon-1',
+          next: null,
+        },
+      },
+    };
+
+    const { result } = renderHook(() => useStepManager(manifest));
+    const { container, rerender } = render(
+      <StepBreadcrumb manifest={manifest} step={result.current.step} />,
+    );
+
+    expect(getAllByRole(container, 'listitem')).toHaveLength(2);
+
+    const labels = container.querySelectorAll('h6.StepBreadcrumb__step__label');
+    const stepsIcons = container.querySelectorAll('div.StepBreadcrumb__step__icon > *');
+    let activeSteps = container.querySelectorAll(
+      'li.StepBreadcrumb__step.StepBreadcrumb__step--active',
+    );
+    let currentStep = container.querySelector(
+      'li.StepBreadcrumb__step.StepBreadcrumb__step--current',
+    );
+    expect(labels).toHaveLength(2);
+    labels.forEach((label, index) => {
+      expect(label).toHaveTextContent(`${index}. Step`);
+    });
```

Comment:
```suggestion
    expect(labels).toHaveLength(2);
    screen.getByRole('heading', { name: `0. Step` })
    screen.getByRole('heading', { name: `1. Step` })
```
As you defined labels just here, we can directly target each label with the defined name without using variables